src/web/components/Content/Content.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Content from './';
```

Comment:
To better match react native components, can you rename it `Text` instead of `Content` ?